drivers/start_date_dq/app/views/start_date_dq/warehouse_reports/reports/_filter.haml

```diff
@@ -1,17 +1,22 @@
 = simple_form_for @filter, as: :filter, url: start_date_dq_warehouse_reports_reports_path, method: :get do |f|
   - content_for :filters_col_full do
+    .row
+      .col-sm
+        = f.input :coc_codes, collection: @filter.coc_code_options_for_select(user: current_user), as: :select_two, input_html: {multiple: true, placeholder: 'Any CoC'}, label: 'CoC Codes'
```

Comment:
@awisema in my local environment, filtering with ALL CoCs selected returns _more_ enrollments than when I filter with no enrollments selected (even though placeholder says "Any CoC"). My guess at what's happening is there is some default CoC(s) selected that are being filtered based on `GrdaWarehouse::Config.get(:site_coc_codes)`. It's not clear to the user that it's happening, though. If that's the case, should I be setting those `site_coc_codes` as the default values here? I see we do that in a few of the CoC selectors, but not most of them.

Reply:
`GrdaWarehouse::Config.get(:site_coc_codes)` goes into `coc_code`, which is used by `filter_for_coc`...

`coc_codes` are set to `filters.dig(:coc_codes)&.select { |code| available_coc_codes&.include?(code) }.presence || coc_codes.presence || user.coc_codes`
and are used by `filter_for_cocs`....

Maybe you have a default CoC code specified on your user?

One way or the other -- this is definitely confusing! Maybe the placeholder should reflect wether there is a default?